### ISSUE TITLE
Query about tracking multiple instances

### ISSUE THREAD

**Issue by dev-aarifsk** [mrousavy/react-native-mmkv#699]
- I came across the following  [comment](https://github.com/Tencent/MMKV/issues/988#issuecomment-1354697634)  by framework dev admonishing not to store mmkv instances as a static variable. is this not true for react-native-mmkv?
- I do want to create a wrapper class that accesses multiple mmkv instances, implementing common functionality for them. if i can't store them as a static variable, how do i track them? 
Thanks

**Comment by mrousavy** [comment]
This is not true for react-native-mmkv, the recommended approach (as mentioned in the README) is to just `export const` the storage instance.
